webapp/channels/src/components/post_view/reaction/reaction.scss

```diff
@@ -1,6 +1,14 @@
 @import 'sass/utils/mixins';
 @import 'sass/utils/functions';
 
+.reaction-emoji--large img {
+    width: 56px;
```

Comment:
Maybe we adjust this to 48x48, since the max resolution is 64 - this might make it a little more crisp on high-res screens

Reply:
Done.